smart_campus/app/models.py

```diff
@@ -227,9 +232,44 @@ def __str__(self):
             category=str(self.category)
         )
 
+    def get_primary_image(self):
+        """Get the primary image
+
+        Returns:
+            image_url (str) if the station primary image exist, None otherwise.
```

Comment:
```
Returns:
    str: Image URL if the primary station image exists, None otherwise.
```

Reply:
If the style below will be much better?
Image URL (str): if the primary station exists
None: otherwise

Reply:
I don't think it would be better.

Maybe you could try PEP 484 type annotation

e.g.
```
from typing import Optional

def the_view() -> Optional[bool]:
    """Function description
    
    Returns:
          return description
    """

```

Since you've annotated at the beginning of the function, you don't have to write it in docstring.